doc/installation-guide.rst

```diff
@@ -220,3 +220,15 @@ file puppetwrapper.properties contains all necessary parameters.
 
 ``#Defaults    requiretty``
 
+
+Known issues
+~~~~~~~~~~~~~~~
+
+   -  When a puppet manifest is executed and the execution of a module fails, in the case where there's more than 1 module installed 
+we don't know wich one has failed -> this information granularity is not provided by puppetdb, so we can't delete the errored 
```

Comment:
errored -> errors ?

I usually do not use there's can't, I write there is and cannot